layouts/_default/baseof.html

```diff
@@ -156,6 +156,15 @@
 
         {{ partial "events.html" (dict "page" . "kind" "sticker") }}
         {{ partial "header.html" . }}
+
+        <!-- Archived Documentation Header -->
+        {{ if and (eq .Section "docs") .Site.Data.args.archive}}
+        <div class="archive-warning-banner" role="alert">
+            <p>⚠️ This documentation is for an older version (<strong>{{ .Site.Data.args.version }}</strong>) and is no longer updated.
+                <a href="/latest{{ .RelPermalink | safeURL }}">Read the latest version</a>.</p>
```

Comment:
sorry for adding yet another hoop but perhaps we should internationalise this text?
Throw the strings in i18n/en.toml and use `{{ i18n "string_desc" }}` to refer to them.  (Plenty of examples to check)

Did you check what happens with this redirect when you're on an archive page that's non-English?